docs/docker.md

```diff
@@ -77,6 +77,8 @@ The monolithic image was split into images for the databases Whyis relies on (_r
 #### Prerequisites 
 * Docker
 * `docker-compose`
+* A data folder in the same directory as your app and Whyis with permissions 777
```

Comment:
should be /data alongside /apps, so I don't think this is right.

Reply:
You're right, just checked.